nginx.montagu.conf

```diff
@@ -71,15 +71,17 @@ server {
         proxy_pass http://contrib/;
     }
     location /reports/ {
-        proxy_pass http://orderly_web_web:8888/;
-        proxy_redirect default;
+        set $orderly_web http://orderly_web_web:8888/;
+        proxy_pass $orderly_web;
+        proxy_redirect $orderly_web /reports/;
```

Comment:
`proxy_redirect default` doesn't work with variables but the above should be equivalent http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_redirect